conference/src/main/java/com/kyriba/conference/management/api/HallController.java

```diff
@@ -111,6 +114,15 @@ void handleEntityNotFound()
   }
 
 
+  // id validation from service level
+  @ExceptionHandler(ConstraintViolationException.class)
```

Comment:
It should work by default, because MVC validation is based on it.

Reply:
yes, it will return 500, but I need 400 here, thats why I remap 'ConstraintViolationException' that comes from HallService when 'id' is not '@Positive'